bin/docker-entrypoint

```diff
@@ -17,7 +17,17 @@ scheduler() {
 
   echo "Starting scheduler and $WORKERS_COUNT workers for queues: $QUEUES..."
 
-  exec /usr/local/bin/celery worker --app=redash.worker --beat -s$SCHEDULE_DB -c$WORKERS_COUNT -Q$QUEUES -linfo --max-tasks-per-child=10 -Ofair
+  /usr/local/bin/celery worker --app=redash.worker --beat -s$SCHEDULE_DB -c$WORKERS_COUNT -Q$QUEUES -linfo --max-tasks-per-child=10 -Ofair
```

Comment:
@rauchy Is this intentionally not using exec to run the worker in production?

Reply:
To be clear, this change leads to the celery worker not receiving signals as explained in https://hynek.me/articles/docker-signals/

Reply:
No, this is a mistake. I initially used the `scheduler` command for both dev and production, but @arikfr suggested I split them and I left this out. Thanks for spotting this!

Reply:
Woot!